client/vat/form-modal/index.tsx

```diff
@@ -24,7 +24,7 @@ const VatFormModal = ( {
 } ): JSX.Element | null => {
 	return isModalOpen ? (
 		<Modal
-			title={ __( 'VAT details', 'woocommerce-payments' ) }
+			title={ __( 'Set your tax details', 'woocommerce-payments' ) }
```

Comment:
Note – I've changed the dialog title for two reasons:

1. Remove `VAT`, not all regions call it `VAT`.
2. Generalise, higher level title. 

Re (2)…

The dialog's purpose is to set various tax details – the tax/VAT number, and address details. It's a multi-step process, so I think this title is a clearer summary of both steps.